sdk/data/aztables/zt_table_recorded_tests.go

```diff
@@ -22,12 +22,12 @@ const (
 
 var ctx = context.Background()
 
-func storageURI(accountName string, endpointSuffix string) string {
-	return fmt.Sprintf("https://%v.table.%v/", accountName, endpointSuffix)
+func storageURI(accountName string) string {
+	return fmt.Sprintf("https://%v.table.core.windows.net/", accountName)
```

Comment:
This may come back to bite you if you ever add sovereign cloud support, but it's fine for now.